### PR TITLE
[KYUUBI #300][TEST] Extract basic jdbc tests to prevent long time UT

### PR DESCRIPTION
**Pull request by turboFei**
turboFei [PR 300](https://github.com/yaooqinn/kyuubi/pull/300) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### Please add issue ID here?

Fixes #300

### Test Plan:
- Add some test cases that check the changes thoroughly including negative and positive cases if possible
- Add screenshots for manual tests if appropriate
- [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by turboFei** [comment]
A little confused about ut failed.

**Comment by yaooqinn** [comment]
aha, seems that you need to change these tests according to master.

**Comment by yaooqinn** [comment]
#292 FYI

**Comment by turboFei** [comment]
got it

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/SharedKyuubiOperationContext.scala [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
-import org.apache.kyuubi.Utils
+import org.apache.kyuubi.{KyuubiFunSuite, Utils}
 import org.apache.kyuubi.config.KyuubiConf
-import org.apache.kyuubi.ha.HighAvailabilityConf._
+import org.apache.kyuubi.ha.HighAvailabilityConf.{HA_ZK_ACL_ENABLED, HA_ZK_QUORUM}
 import org.apache.kyuubi.ha.server.EmbeddedZkServer
 import org.apache.kyuubi.server.KyuubiServer
 
-abstract class KyuubiOperationSuite extends JDBCTests {
+trait SharedKyuubiOperationContext extends KyuubiFunSuite {
```
Comment (yaooqinn) [inline kyuubi-main/src/test/scala/org/apache/kyuubi/operation/SharedKyuubiOperationContext.scala:26]: For sense of symmetry, `WithKyuubiServer` as we have a `WithSparkSQLEngine` 😺

Comment (turboFei) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/SharedKyuubiOperationContext.scala:26]: 👌

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.1.0
